src/plugins/inappbrowser.ts

```diff
@@ -12,76 +13,72 @@ export interface InAppBrowserEvent extends Event {
   message: string;
 }
 
```

Comment:
Could you add a quick comment on very basic usage of this? The instance method stuff is a bit new for people and we haven't talked a ton about it.